### PR TITLE
Feature: Update App Documentation / README

### LINKED ISSUE #3470: Feature: Improve README / Documentation
**Why?**

There are a number of useful tools and guides for working with the app that could be highlighted better. We could also provide more information and context about the web app itself, rather than general Odin information which is covered well in a number of other places.

**How?**

- [ ] Highlight useful tools for setting up and working on the app
  - [ ] `bin` scripts
    - [ ] test
    - [ ] update / setup
    - [ ] dev
  - [ ]  Lookbook
  - [ ] Lesson preview tool
- [ ] Add overview of tech stack
- [ ] Add overview of sites / tools
- [ ] Add links to wiki
- [ ] Add quickstart guide for more experienced devs

### PR DESCRIPTION
**Pull request by ChargrilledChook**
## Because
WIP. Putting up early To get early feedback on overall direction

## This PR
- Updates README.md with app specific info and useful links
- Updates `env.sample` 

## Issue
Closes #3470

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,78 @@
...
+- Sidekiq
+- Kramdown
+
+### Testing and Linting
+- RSpec
+- Jest
+- Rubocop
+- Erblint
+- ESLint
+
```
Comment (ChargrilledChook) [inline README.md:51]: Seeing it now I feel like this is overkill, at least for the README. Perhaps better suited to the wiki or another doc.

File: README.md [hunk 2]
```diff
@@ -1,54 +1,78 @@
...
--   [Rachel Moser](https://github.com/rlmoser99)
--   [Grace Schwartz](https://github.com/CatQueenCodes)
--   [Tatiana Walton](https://github.com/twalton83)
--   [Valera Kushnir](https://github.com/kashura)
-
----
-
-Created by [Erik Trautman](http://www.github.com/eriktrautman)
+## Significant Contributors
+_This should be kept, but should it be directly in the README or linked to from here?_
```
Comment (ChargrilledChook) [inline README.md:78]: Thoughts? Should this be kept as is? Put into `significant_contributors.md` and linked to? Something else entirely? There's a few spots where this information is duplicated somewhat, especially with the hall of fame

File: env.sample [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
 ODIN_BOT_ACCESS_TOKEN: ''
 
-GITHUB_API_TOKEN: 1234
-GITHUB_APP_ID: 1234
-GITHUB_SECRET: 1234
-GITHUB_WEBHOOK_SECRET: 1234
+# Google Omniauth
 GOOGLE_CLIENT_ID: 1234
 GOOGLE_CLIENT_SECRET: 1234
-SPAM_DOMAINS: 'example.com, baddomain.com'
```
Comment (ChargrilledChook) [inline env.sample:17]: Couldn't see this referenced anywhere so took it out.

**Review by ChargrilledChook — COMMENTED** [review]
Obviously still WIP, but welcome any feedback / suggestions at this point. Overally shape of it is largely there, I think.

File: README.md (hunk 1 above)
Comment (KevinMulhern) [reply README.md:51]: I can see where you're coming from by feeling it's overkill. I can't think of another spot for it though 😅 
There would be no harm in leaving it here imo

File: README.md (hunk 2 above)
Comment (KevinMulhern) [reply README.md:87]: We have an idea in the backlog for [creating a team page](https://github.com/TheOdinProject/top-meta/issues/3) which may be a nice replacement/ solution to all this duplication.

File: README.md (hunk 1 above)
Comment (JuanVqz) [reply README.md:51]: What about moving it to the CONTRIBUTING file under a new `Tech we use` section?

**Comment by scheals** [comment]
@ChargrilledChook RE: running all the tests and such from the PR where you mentioned `bin/test`, perhaps this bit of [Contributing.md](https://github.com/TheOdinProject/theodinproject/wiki/Contributing-Guide#how-to-contribute) can be changed to that:
image

**Comment by ChargrilledChook** [comment]
Still planning on doing this but need to revisit, many suggestions / changes have been implemented in the Wiki

**Pull request closed without merging**